include/control_toolbox/dither.h

```diff
@@ -71,6 +72,19 @@ class Dither
    */
   double update();
 
+   /*
+   *\brief Dither gets an amplitude, must be >0 to initialize.
+   *
+   * Initialize random number generator with random_device for non-deterministic random numbers
+   *
+   *\param amplitude Amplitude of white noise output
+   */
+  bool init(const double &amplitude)
```

Comment:
I'd prefer a static function to generate a non-deteministic seed.
This way user code would stay the same for use is test and production.

Reply:
```c++
Dither dither;
double seed = Dither::generateRandomSeed();
ROS_INFO_STREAM("Use seed " << seed);
dither.init(1.0, seed);
```

Reply:
> ```c++
> Dither dither;
> double seed = Dither::generateRandomSeed();
> ROS_INFO_STREAM("Use seed " << seed);
> dither.init(1.0, seed);
> ```

taking this as an example, sent out updated version